lib/t/calls.ex

```diff
@@ -322,42 +322,33 @@ defmodule T.Calls do
     Multi.new()
     |> Multi.run(:match, fn _repo, _changes ->
       match =
-        Matches.matches_with_undying_events_q()
+        Match
         |> where(id: ^match_id)
         |> where([m], m.user_id_1 == ^caller_id or m.user_id_2 == ^caller_id)
-        |> select([m, undying_event: e], {m.inserted_at, [m.user_id_1, m.user_id_2], e.timestamp})
+        |> select([m], {[m.user_id_1, m.user_id_2]})
         |> Repo.one()
 
       case match do
-        {inserted_at, users, undying_event_at} ->
+        {users} ->
```

Comment:
```suggestion
        users ->
```

Reply:
I think it might have been simpler before I added the timer logic, I'll check it out tomorrow.

Reply:
https://github.com/getsince/test3/pull/479/commits/27d5d2ed0529d00ae0ec1615a82b3fe47a1b6554